src/app/table-module/containers/home-page/home-page.component.ts

```diff
@@ -0,0 +1,106 @@
+
+import { Component, OnInit, Inject } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+import { GridComponent, GridDataResult, CancelEvent, EditEvent, RemoveEvent, SaveEvent, AddEvent } from '@progress/kendo-angular-grid';
+import { State, process } from '@progress/kendo-data-query';
+import { Product } from './model';
+import { EditService } from './edit.service';
+import { map } from 'rxjs/operators';
+
+
+@Component({
+  selector: "app-home-page",
+  templateUrl: "./home-page.component.html",
+  styleUrls: ["./home-page.component.scss"],
+})
+export class HomePageComponent {
+   public view: Observable<GridDataResult>;
+    public gridState: State = {
+        sort: [],
+        skip: 0,
+        take: 5
+    };
+    public formGroup: FormGroup;
+
+    private editService: EditService;
+    private editedRowIndex: number;
+
+    constructor(@Inject(EditService) editServiceFactory: () => EditService) {
```

Comment:
way you use to inject dependency is not correct.

Reply:
remove the factory method